internal/services/synapse/synapse_sql_pool_resource.go

```diff
@@ -181,7 +206,13 @@ func resourceSynapseSqlPool() *pluginsdk.Resource {
 			"tags": commonschema.Tags(),
 		},
 
-		CustomizeDiff: pluginsdk.CustomizeDiffShim(synapseSqlPoolCustomizeDiff),
+		CustomizeDiff: pluginsdk.CustomDiffWithAll(
+			pluginsdk.CustomizeDiffShim(synapseSqlPoolCustomizeDiff),
+			// Maintenance schedule can't be removed once set
+			pluginsdk.ForceNewIfChange("maintenance_schedule", func(ctx context.Context, old, new, meta interface{}) bool {
+				return len(old.([]interface{})) > 0 && len(new.([]interface{})) == 0
```

Comment:
So removing the maintenance schedule will force-new, but changing the schedule is allowed?